spec/requests/api/additional_document_validation_request_spec.rb

```diff
@@ -30,7 +30,7 @@
             {
               "id" => additional_document_validation_request.id,
               "state" => "open",
-              "response_due" => additional_document_validation_request.response_due.to_s,
+              "response_due" => additional_document_validation_request.response_due.to_s(:db),
```

Comment:
ohh what does the :db thing to? never seen it before, how would one know how to use it?

Reply:
so the API doesn't format the data it sends, so it send the "raw" timestamps (as opposed to our prettification of it), which are in the `db` format: https://railsdatetimeformats.com/